Mathlib/NumberTheory/RamificationInertia.lean

```diff
@@ -447,9 +446,13 @@ noncomputable instance Quotient.algebraQuotientPowRamificationIdx : Algebra (R 
   Quotient.algebraQuotientOfLEComap (Ideal.map_le_iff_le_comap.mp le_pow_ramificationIdx)
 #align ideal.quotient.algebra_quotient_pow_ramification_idx Ideal.Quotient.algebraQuotientPowRamificationIdx
 
+-- Adaptation note: 2024-04-23
+-- The right hand side here used to be `Ideal.Quotient.mk _ (f x)`, but this is now slow without
+-- `set_option backward.isDefEq.lazyProjDelta false in`
+-- Instead we've replaced it with `Ideal.Quotient.mk (P ^ e) (f x)` (compare #12412)
```

Comment:
Just a comment: This was already slow previously, only not *that* slow. (Similarly in the analogous case below, where it was not quite as slow as here.)